### PR TITLE
[Backport 1.11 ] fix(tasknametextfilter): include id search in task text filter

### PR DESCRIPTION
**Pull request by bstavroulakis**
BACKPORT Include ID search in task text filter.

**Checklist**
- [ ] Did you add a JIRA issue in a commit message or as part of the branch name?
- [ ] Did you add new unit tests?
- [ ] Did you add new integration tests?
- [ ] If this is a regression, did you write a test to catch this in the future?

### CODE REVIEW

File: plugins/services/src/js/filters/TaskNameTextFilter.js [hunk 1]
```diff
@@ -23,7 +23,10 @@ class TaskNameTextFilter extends DSLFilter {
    */
   filterApply(resultSet, filterType, filterArguments) {
     return resultSet.filterItems(task => {
-      return task.name.indexOf(filterArguments.text) !== -1;
+      return (
+        (task.id && task.id.includes(filterArguments.text)) ||
```
Comment (juliangieseke) [inline plugins/services/src/js/filters/TaskNameTextFilter.js]: ❤️  - can you please forward port this to `master` :trollface:

**Comment by bstavroulakis** [comment]
run integration test

**Comment by bstavroulakis** [comment]
run system tests

**Comment by bstavroulakis** [comment]
run integration tests

**Comment by ahoskins** [comment]
run integration tests

**Pull request merged**